scripts/supercloud/run_pg3_tamp_experiments.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+FILE="scripts/supercloud/submit_supercloud_job.py"
+ALL_NUM_TRAIN_TASKS=(
+     "10"
+     "50"
+     "100"
+)
+ENVS=(
+    "cover"
+    "painting"
+    "screws"
+    "repeated_nextto"
+    "cluttered_table"
+    "coffee"
+)
+
+for ENV in ${ENVS[@]}; do
+  for NUM_TRAIN_TASKS in ${ALL_NUM_TRAIN_TASKS[@]}; do
+
+      if [ $ENV = "painting" ] || [ $ENV = "cluttered_table" ] || \
+          [ $ENV = "coffee" ] then
+        COMMON_ARGS="--env $ENV  --strips_learner oracle \
+              --sampler_learner oracle --num_train_tasks $NUM_TRAIN_TASKS \
+              --pg3_hc_enforced_depth 1"
```

Comment:
I'm thinking we should just use depth 1 for everything? It's generally better if we can use one set of hyperparameters for all environments and approaches. We would just expect it to take longer to finish, right? And if we want to later, we could even optimize this by terminating as soon as we find a policy with heuristic 0.

Reply:
Oh wait you're right because for the easier environments they won't even need to look at that depth. I was just avoiding complexity when evaluating the easier environments, but yeah it won't matter.